.env.test

```diff
@@ -11,3 +11,7 @@ EXPORT_DATE_OVERRIDE=
 EPOCH_TIMESTAMP=1568059200
 
 ROLLBAR_TOKEN=
+
+ROLLBAR_TOKEN_FILE=rollbar_token
+DB_CRED_FILE=db_pass
+DB_USER_FILE=db_user
```

Comment:
what's the thinking behind 2 separate files vs a single one that looks like
```yaml
# doesn't have to be yaml
username: user
password: password
```

Reply:
Based upon the reading here: https://kubernetes.io/docs/concepts/configuration/secret/#consuming-secret-values-from-volumes

Reply:
https://github.com/getoutreach/cdc_snapshot_validation/blob/master/deployments/aurora-bootstrap.jsonnet#L125-L127

Reply:
sold. sounds good